### DISCUSSION TITLE
Start actions manually from app

### DISCUSSION THREAD

**Discussion by menjoo** [community/community#130712] [Actions]
It would be really helpful if actions could be started from within the app. Currently one can only view actions.

**Comment by joshjohanning** [comment]
You can use the CLI, at least, if you are using the `workflow_dispatch` trigger. :)

```yml
gh workflow run workflow.yml -R joshjohanning-org/actions-throwaway
```

The `-R` param is not needed if you are running this command in the cloned repo.
